### PR TITLE
Hotfix 15.0.1

### PR DESCRIPTION
**Pull request by dochoaj**
Fix asset pipeline on module generation.

### CODE REVIEW

File: module/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = yeoman.generators.Base.extend({
 
   addModuleToApplication: function () {
     var application = this.readFileAsString('application.coffee');
-    application = application + '\n' + '#= require_tree modules/' + this.module_filename;
+    application = application + '\n' + '#= require modules/' + this.module_filename + '/' + this.module_filename + '.coffee';
```
Comment (Ajeo) [inline module/index.js]: @dochoaj can you split this line to improve the readability?

Comment (dochoaj) [inline module/index.js]: @Ajeo No problem bro

**Comment by dochoaj** [comment]
@Ajeo Ready, please review again.

**Comment by Ajeo** [comment]
:+1:

**Pull request merged**